scala-conductr-client-lib/src/main/scala/com/typesafe/conductr/clientlib/scala/models/BundleRunResult.scala

```diff
@@ -0,0 +1,32 @@
+package com.typesafe.conductr.clientlib.scala.models
+
+import java.util.UUID
+
+import play.api.libs.json.{ Format, Json }
+
+sealed trait BundleRunResult {
+
+  def isSuccess: Boolean = this.isInstanceOf[BundleRunSuccess]
+  def isFailure: Boolean = this.isInstanceOf[BundleRunFailure]
+
+  def fold[T](failure: BundleRunFailure => T, success: BundleRunSuccess => T): T = this match {
+    case s: BundleRunSuccess => success(s)
+    case f: BundleRunFailure => failure(f)
+  }
+}
+
+case class BundleRunSuccess(
+  requestId: UUID,
+  bundleId: BundleId) extends BundleRunResult
+
+object BundleRunSuccess {
+  implicit val bundleRunSuccessFormat: Format[BundleRunSuccess] = Json.format
+}
+
+case class BundleRunFailure(
+  code: Int,
+  error: String) extends BundleRunResult
```

Comment:
We can probably generalise failure results if all that is being returned is an error string.